pkgs/development/tools/build-managers/msbuild/create-deps.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p msbuild
+set -euo pipefail
+
+cat << EOL
+{ fetchurl }: [
+EOL
+
+tmpdir="$(mktemp -d -p "$(pwd)")" # must be under source root
+trap 'rm -rf $tmpdir' EXIT
+
+(
+    ulimit -n 8192 # https://github.com/NuGet/Home/issues/8571
+    export HOME="$tmpdir"
+    msbuild -noAutoRsp -p:Configuration=Release-MONO -t:restore -p:RestoreNoCache=true MSBuild.sln
+    msbuild -noAutoRsp -p:RestoreNoCache=true mono/build/update_bundled_bits.proj
+    msbuild -noAutoRsp -t:restore -p:RestoreNoCache=true "$tmpdir"/.nuget/packages/microsoft.dotnet.arcade.sdk/*/tools/Tools.proj
+    msbuild -noAutoRsp -p:Configuration=Release-MONO -p:RepoRoot="$(pwd)" -p:Restore=true -p:RestoreNoCache=true "$tmpdir"/.nuget/packages/microsoft.dotnet.arcade.sdk/*/tools/Build.proj
+) | \
+    sed -nr 's/^ *OK *(.*\.nupkg).*$/\1/p' | \
+    sort -u | \
+    while read url; do
+        sha256=$(nix-prefetch-url "$url" 2>/dev/null)
+        cat << EOL
+  (fetchurl {
+    url = "$url";
+    sha256 = "$sha256";
+  })
+EOL
+    done
+
+cat << EOL
+]
+EOL
```

Comment:
```suggestion
echo -n "]"
```

Reply:
I thought about this, but the way it is is at least consistent with the header/entry output.  Also it's meant to have the trailing \n